pdf/src/campPrint/program/ProgramPeriod.vue

```diff
@@ -25,10 +25,7 @@ export default {
   },
   computed: {
     sortedScheduleEntries() {
```

Comment:
I think we should rename the computed to `scheduleEntries`. Or add a comment, that the scheduleEntries are already sorted. Otherwise this seems confusing to readers.

Reply:
[05a2af5](https://github.com/ecamp/ecamp3/pull/4600/commits/05a2af534164146ea3e27424688e0f47b845a553)